keylime-agent/src/config.rs

```diff
@@ -515,6 +532,14 @@ impl Source for KeylimeConfig {
             "agent_data_path".to_string(),
             self.agent.agent_data_path.to_string().into(),
         );
+        _ = m.insert(
```

Comment:
So many `_` seems a bit ugly. I wonder if this can help: https://github.com/rust-lang/rfcs/issues/3092

Reply:
Thanks for the pointer, but unfortunately we cannot use this (yet). It is still available only on nightly:
https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.try_insert